Packet++/src/DnsResourceData.cpp

```diff
@@ -61,17 +61,19 @@ IPv4DnsResourceData::IPv4DnsResourceData(const uint8_t* dataPtr, size_t dataLen)
 	}
 
 	uint32_t addrAsInt = *(uint32_t*)dataPtr;
-	m_Data = IPv4Address(addrAsInt);
```

Comment:
here the behavior changes. Previously we didn't guarantee the correctness of the IP address, but now the IP address cannot be invalid.